app/PaymentDrivers/StripePaymentDriver.php

```diff
@@ -233,6 +235,13 @@ public function gatewayTypes(): array
             && in_array($this->client->country->iso_3166_3, ['CAN', 'USA'])) {
             $types[] = GatewayType::ACSS;
         }
+        if ($this->client
+            && $this->client->currency()
+            && in_array($this->client->currency()->code, ['EUR', 'USD', 'GBP', 'DKK', 'SEK', 'NOK'])
+            && isset($this->client->country)
+            && in_array($this->client->country->iso_3166_3, ['DE', 'AT', 'BE'])) {
```

Comment:
You are scanning for iso_3166_3 which means it is the 3 character representation, you'l want to cover these countries.

Austria, Belgium, Denmark, Finland, France, Germany, Ireland, Italy, Netherlands, Norway, Spain, Sweden, United Kingdom, and the United States

```
'AUT','BEL','DNK','FIN','FRA','DEU','IRL','ITA','NLD','NOR','ESP','SWE','GBR','USA'
```

Reply:
Solved